pkg/util/removeall/removeall_test.go

```diff
@@ -236,11 +236,11 @@ func TestRemoveDirsOneFilesystem(t *testing.T) {
 		for _, item := range test.items {
 			if strings.HasSuffix(item, "/") {
 				item = strings.TrimRight(item, "/")
-				if err = os.Mkdir(path.Join(tmpDir, item), 0777); err != nil {
+				if err = os.Mkdir(filepath.Join(tmpDir, item), 0777); err != nil {
```

Comment:
the test items all use `/`... does filepath.Join translate that to `\` on windows or do we end up with mixed slashes?